runhouse/resources/packages/package.py

```diff
@@ -433,14 +423,14 @@ def split_req_install_method(req_str: str):
 
     @staticmethod
     def from_config(config: dict, dryrun=False, _resolve_children=True):
-        if isinstance(config.get("install_target"), dict):
-            from runhouse.resources.folders import Folder
-
-            config["install_target"] = Folder.from_config(
-                config["install_target"],
-                dryrun=dryrun,
-                _resolve_children=_resolve_children,
-            )
+        # if isinstance(config.get("install_target"), dict):
```

Comment:
can we remove now?

Reply:
actually... I need to mess with the whole `.config()`. good call.

Reply:
fixed now!